sql_metadb/derived_tables/instance_ext.sql

```diff
@@ -22,8 +22,8 @@ SELECT
     jsonb_extract_path_text(i.jsonb, 'modeOfIssuanceId')::uuid AS mode_of_issuance_id,
     moit.name AS mode_of_issuance_name,
     jsonb_extract_path_text(i.jsonb, 'catalogedDate')::date AS cataloged_date,
-    jsonb_extract_path_text(i.jsonb, 'metadata', 'createdDate')::date AS created_date,
-    jsonb_extract_path_text(i.jsonb, 'metadata', 'updatedDate')::date AS updated_date
+    jsonb_extract_path_text(i.jsonb, 'metadata', 'createdDate')::timestamptz AS created_date,
```

Comment:
The `createdDate` data do not appear to have a timezone.  It should probably have the `timestamp` data type.